crates/matrix-sdk/src/event_cache/room/mod.rs

```diff
@@ -213,6 +213,18 @@ impl RoomEventCache {
         RoomPagination { inner: self.inner.clone() }
     }
 
+    /// Try to find a single event in this room, starting from the most recent
+    /// event.
+    ///
+    /// **Warning**! It looks into the loaded events in `RoomEvents`
```

Comment:
Consider using ```[`RoomEvents`]``` here, so the renaming in my PR would have become a doc build error here :)

(or better, don't use the type name, and rephrase as: "it looks into the in-memory linked chunk only"

Reply:
Done.